pkg/rbd/rbd_util.go

```diff
@@ -613,6 +619,40 @@ func genVolFromVolumeOptions(ctx context.Context, volOptions, credentials map[st
 	return rbdVol, nil
 }
 
+// validateAndGetImageFeaturesWithMounter validates image features and mounters, and gets their values
+func validateAndGetImageFeaturesWithMounter(volOptions map[string]string) (string, string, error) {
+	mounter, found := volOptions["mounter"]
+	if !found {
+		mounter = rbdDefaultMounter
+	}
+
+	imageFeatures, found := volOptions["imageFeatures"]
+	if !found {
+		return "", mounter, nil
+	}
+
+	arr := strings.Split(imageFeatures, ",")
```

Comment:
This is probably outside the scope of the PR, but why split strings on a delimiter. Isn't the data coming in thru yaml, as structured data? Can't this be a proper list/slice before it gets here?

Reply:
We have imageFeatures as String in storageclass before, here see [link](https://github.com/rook/rook/blob/master/cluster/examples/kubernetes/ceph/csi/rbd/storageclass.yaml#L39). It would be nice to change to yaml's structured data, but it would be difficult because it is not compatible with the previous version.

Reply:
yeah, the imagefeature in storage-class is a comma-separated values